### PR TITLE
Update pipeline.py

### PR DESCRIPTION
**Pull request by bhavna3**
The modification replaces this method with a token-based approach using a tokenizer. This tokenizes the entire prompt, identifies the token corresponding to the trigger word, removes it, and then decodes the modified tokens back into text.

### CODE REVIEW

File: photomaker/pipeline.py [hunk 1]
```diff
@@ -344,7 +344,15 @@ def __call__(
         )
         
         # 4. Encode input prompt without the trigger word for delayed conditioning
-        prompt_text_only = prompt.replace(" "+self.trigger_word, "") # sensitive to white space
```
Comment (PublisherApplications) [inline photomaker/pipeline.py:347]: These changes are perfect, I like how the way you remove stuff and you add more stuff, If I was a commitment developer I will love to make you a collaborator or whatever it is.

File: photomaker/pipeline.py [hunk 2]
```diff
@@ -490,4 +498,4 @@ def __call__(
         if not return_dict:
             return (image,)
 
-        return StableDiffusionXLPipelineOutput(images=image)
```
Comment (PublisherApplications) [inline photomaker/pipeline.py:493]: you eattttttt ) I do not mind that but you did a good job.

**Review by PublisherApplications — APPROVED** [review]
All your changes are perfect All we have to do is wait until a collaborator approves it or merge it. Based on these changes I think you're going to get merge.

File: photomaker/pipeline.py (hunk 1 above)
Comment (bhavna3) [reply photomaker/pipeline.py:347]: Thanks a lot! Glad you liked the changes. Excited to contribute more in the future!

**Pull request merged**